### PR TITLE
add link to feedback and suggestions

### PR DESCRIPTION
**Pull request by MollieCourtenay**
for HEALTHLOCKER, add link to feedback page to the bottom of the stories page.

#896

### CODE REVIEW

File: web/templates/post/index.html.eex [hunk 1]
```diff
@@ -12,3 +12,5 @@
     </article>
   <% end %>
 </section>
+<p><%= link "Write your own story, or suggest an addition or amendment to this page.", to: "/feedback", class: "hl-aqua flex-wrap underline" %></p>
```
Comment (SimonLab) [inline web/templates/post/index.html.eex]: I think this paragraph should be placed inside the section (just one line up)

**Comment by MollieCourtenay** [comment]
@SimonLab done it

**Comment by MollieCourtenay** [comment]
@SimonLab  Ok. Who'll do that? Should that be me?

**Comment by SimonLab** [comment]
It's a very small change, @RobStallion @katbow do you mind to check the indentation on one of your next PRs?

**Pull request merged**